examples/plot_gait.py

```diff
@@ -173,24 +162,25 @@
 mc_fig.legend(loc="upper center")
 
 
-""
-X_mc = trial['data']
+###############################################################################
+# Let's put the data in shape for DiCoDiLe
 
-""
-X_mc_plain = rfn.structured_to_unstructured(X_mc).T
+X_mc_subset = trial['data'][channels].to_numpy().T
+X_mc_subset.shape
```

Comment:
To display this in the example, we need to add a `print` statement:
```suggestion
print(X_mc_subset.shape)
```